pom.xml

```diff
@@ -10,7 +10,7 @@
   <artifactId>plugin</artifactId>
   <!-- Baseline Jenkins version you use to build and test the plugin. Users 
    must have this version or newer to run. -->
-  <version>2.8</version>
+  <version>1.609.3</version>
```

Comment:
I understand, but I'd preferred to lower the required Jenkins version if 2.8 isn't a must - the commend above 'Users must have this version or newer to run.'.

I've started to test and run my recent baked version of the plugin against 1.609.3 (LTS) and no issue found yet related to lowering the requested Jenkins version.